sirepo/auth_db.py

```diff
@@ -310,6 +310,7 @@ def uids_of_paid_users(cls):
 
     class UserRoleInvite(UserDbBase):
         __tablename__ = 'user_role_invite_t'
+        email = sqlalchemy.Column(UserDbBase.STRING_NAME, primary_key=True)
```

Comment:
We don't need to add this field to the table since the data is already in auth_email_user_t we could join on uid with it and get it if we ever need it from there.

Reply:
@e-carlin should I do the join in api_getModerationRequestRows then?

Reply:
e49bbdccbaf2d849005479713d0b0f6cc190d19b